analytics/events.ts

```diff
@@ -0,0 +1,7 @@
+export const EVENT_NAMES = {
+  INPUT: `Input mode selected`,
+  SUBMIT: "Submitted request",
+  LENGTH: `Length selected`,
+  SHARED: "Share button clicked",
+  NEW_SUMMARY: "New summary button clicked",
+};
```

Comment:
Do you think we should add an event for the Results page (like RESULT_VIEWED) too? that way we can ensure if the users are reaching the results page after the SUBMIT event and not getting stuck in the loading page

Reply:
Good idea!

Reply:
I added the results, I added better error handling and analytics for the errors as well!

Reply:
@MaiaS 🔥🔥